setup.py

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env python
 
 from setuptools import setup, find_packages
+
 import provider
 
 setup(
-    name='django-oauth2-provider',
+    name='edx-django-oauth2-provider',
     version=provider.__version__,
-    description='Provide OAuth2 access to your app',
+    description='edX fork of django-oauth2-provider',
     long_description=open('README.rst').read(),
-    author='Alen Mujezinovic',
-    author_email='alen@caffeinehit.com',
-    url = 'https://github.com/caffeinehit/django-oauth2-provider',
-    packages= find_packages(exclude=('tests*',)),
+    author='edX',
```

Comment:
The original author is no longer maintaining this package, which an
edX-modified fork. If folks want to track him down they still can; but, any
questions or concerns about this specific package should be addressed to
edX, not him.

-- 
Clinton Blackburn